src/Components/SideMenu.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { BrowserRouter, Route, Link } from 'react-router-dom'
+import MenuList from '@material-ui/core/MenuList'
+import MenuItem from '@material-ui/core/MenuItem'
+import { makeStyles } from '@material-ui/core/styles'
+import IconAbout from '../Assets/icon-about.png'
+import IconMoney from '../Assets/icon-money.png'
+import IconCountry from '../Assets/icon-country.png'
+import IconPerson from '../Assets/icon-person.png'
+import LuisCoin from '../Views/LuisCoin'
+import LuisCountries from '../Views/LuisCountries'
+import ChangedLifes from '../Views/LifesLuisChanged'
+import AboutLuis from '../Views/AboutLuis'
+
+const useStyles = makeStyles({
+    root: {
+        position: 'fixed',
+        top: '50%',
+        transform: 'translateY(-50%)',
+    },
+    iconBG: {
+        background: "linear-gradient(93.37deg, #00EC5E 0%, #00A3FF 100%)",
+        borderRadius: "100px",
+        boxShadow: "1px 1px 6px rgba(0, 0, 0, 0.25)",
+        height: '50px',
+        margin: "10px 0 10px 0",
+        position: 'relative',
+        width: '50px',       
+    },
+    icon: {
+        height: '30px',
+        left: '20%',
+        position: 'absolute',
```

Comment:
Não é bom utilizar position absolute, pois no momento que for adicionado o texto o ícone vai ficar por cima dele. Pode só tirar o position e colocar _margin_ no lugar de _left_ e _top_